packages/plugin-exec/sources/ExecFetcher.ts

```diff
@@ -86,6 +98,7 @@ export class ExecFetcher implements Fetcher {
   private async generatePackage(cwd: PortablePath, locator: Locator, generatorPath: PortablePath, opts: FetchOptions) {
     return await xfs.mktempPromise(async binFolder => {
       const env = await scriptUtils.makeScriptEnv({project: opts.project, binFolder});
+      const envFile = ppath.join(binFolder, `environment.js` as Filename);
```

Comment:
The `buildFolder` directory is in the PATH; let's rather use `cwd`, which is perfect for that since it's the temporary directory for the generator.